components/rewards/RewardsFooter.js

```diff
@@ -0,0 +1,76 @@
+import { FontAwesome5 } from '@expo/vector-icons';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { View } from 'react-native';
+import { Subhead } from '../BaseComponents';
+
+/**
+ * @prop
+ * */
+
+function RewardsFooter({ wdth, customer, isGuest }) {
+  // const points = parseInt(customer.points);
+  let string1 = '';
+  let string2 = 'Your next reward >';
+  if (customer) {
+    string1 = `${customer.points} points`;
+  }
+  if (isGuest) {
+    string1 = 'Learn about healthy rewards >';
+    string2 = '';
+  }
+  if (customer) {
```

Comment:
Hmm to avoid repeating too much code, I would recommend replacing lines 22-62 with conditional rendering! https://reactjs.org/docs/conditional-rendering.html
Logical && operator sounds like what you might be going for with lines 37-41 since that's virtually the only major difference between the two returns